### PR TITLE
JavaScript/testing_javascript/more_testing.md - move Jest setup to Additional Resources

### LINKED ISSUE #26548: JavaScript/More_Testing: Emphasize that setting up jest for webpack is not required
### Describe your suggestion

I've seen a commonly asked question where in the lesson [More Testing](https://www.theodinproject.com/lessons/node-path-javascript-more-testing) assignement 6 describes:

> And finally, if you wish, you can add Jest to your webpack setup. Read about that process [here](https://jestjs.io/docs/webpack).

Problem: Even though we already have completed projects where we only use jest/babel to test it still gives an impression that we still need to setup jest for webpack in the following project, but this is not the case because like the documentation describes, it's useful for handling static assets, css modules, etc in tests. These things are not necessary in fully testing the modules in the following project Battleship. It is also encouraged to create the project and create the modules without handling any kind of DOM manipulation as a good practice, so we won't really see any issues coming up even if they ignored this assignment.

Solution: Emphasize that setting up jest for webpack is not necessary. This I think could also be more of an additional resource than an assignment (IMO)

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-more-testing

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### (Optional) Discord Name

nevz

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by aconstas**
## Because

Emphasize that setting up Jest is not completely necessary for this project. However, provide the resource later if users wish to.

## This PR

- Moved "If you wish, you can add Jest to your webpack setup. Read about that process [here](https://jestjs.io/docs/webpack)" from "Assignment" to "Additional Resources"

## Issue

Closes #26548 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/testing_javascript/more_testing.md [hunk 1]
```diff
@@ -91,3 +90,5 @@ This section contains questions for you to check your understanding of this less
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
 1. This [jest cheat sheet](https://github.com/sapegin/jest-cheat-sheet) walks you quickly through all the frequently used methods.
+
```
Comment (KevinMulhern) [inline javascript/testing_javascript/more_testing.md:93]: Can this new line be removed please?

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

File: javascript/testing_javascript/more_testing.md (hunk 1 above)
Comment (aconstas) [reply javascript/testing_javascript/more_testing.md]: Done!

**Review by KevinMulhern — APPROVED** [review]
Thanks @aconstas 💪

**Comment by aconstas** [comment]
@KevinMulhern Thank you!🚀

**Pull request merged**